src/Monolog/Formatter/LogmaticFormatter.php

```diff
@@ -11,6 +11,10 @@
 
 namespace Monolog\Formatter;
 
+const APP_FIELD_NAME = "appname";
+const HOST_FIELD_NAME = "hostname";
+const MARKER_FIELD_NAME = "@marker";
```

Comment:
Please define these as class constants if anything, although I don't know that changing existing code for the sake of changing it brings much value. But anyway they can't be just defined as regular constants as it might clash with other stuff.